lib/kitchen/busser.rb

```diff
@@ -40,23 +56,10 @@ class Busser
     # @option opts [String] :kitchen_root local path to the root of the project
     # @option opts [String] :instance_ruby_bindir path to the directory
     #   containing the Ruby binary on the remote instance
-    # @option opts [TrueClass, FalseClass] :sudo whether or not to invoke
-    #   sudo before commands requiring root access (default: `true`)
```

Comment:
I'm assuming you are referring to dropping sudo?

I don't think it does, but that could be open to interpretation. The former code enforced two separate sudo configurations. One here in the busser and another in the provisioner. I think it is more clear to have one and if there are kitchen.yml files out there that specify sudo in a busser config, it will simply be ignored.

My guess is that if sudo was specified for the busser it was also specified for the provisioner. I could be wrong and others may have more insight, but i *think* this is safe.